### DISCUSSION TITLE
Can't get my posts to show up on lab notebook link

### DISCUSSION THREAD

**Discussion by drlawson** [RobertsLab/resources#1263] [Q&A] [answered]
Hi all, I made a lab notebook using the Jekyll instructions in the lab handbook, and I was able to edit the original post but now I can't get my second post to show up on the website. Let me know if anyone knows what I did wrong or if you need any more information. Thanks! :)

https://github.com/drlawson/drlawson.github.io

https://drlawson.github.io/

**Comment by aspencoyle** [comment, marked as answer]
Ooh okay I ran into a similar issue a while ago! The file name of your post needs to be YYYY-MM-DD-postnamewhateverhere.extension

The last dash after the date is also important - I accidentally had a post or two named something like YYYY-MM-DD_postname and they wouldn't show up until I changed that underscore to a dash

**Reply by drlawson** [reply]
That worked perfectly thank you so much!!
